prow/ssm-agent/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ssm-DaemonSet.yaml
```

Comment:
nit: lowercase `ssm-daemonset.yaml`

Reply:
Kept it inline with the templates in the other Helm charts - https://github.com/aws-controllers-k8s/test-infra/tree/main/prow/data-plane/templates

Reply:
meh, ok :)